tests/integration/session/test_login.py

```diff
@@ -365,6 +366,25 @@ def test_login_token_with_invalid_cir_instrument_id_results_in_500(self):
         # Then
         self.assertException()
 
+    @patch("app.routes.session.get_supplementary_data_v1")
+    @patch("app.routes.session._validate_supplementary_data_lists")
+    @patch(
+        "app.questionnaire.questionnaire_store_updater.QuestionnaireStoreUpdaterBase.set_supplementary_data",
+    )
+    def test_login_with_sds_schema_version_valid(
```

Comment:
You probably need to move your testing to `test_supplementary_data_parser.py` where the class and methods you changed are tested.

Reply:
The purpose of this test was just to check if the schema opens, I initially did a functional test, but then we decided it would be better to add it as an integration test. So we chose to put it in test_login as these just check if schemas can open?

Reply:
I'm not sure that this is testing anything at the moment though cause if you change the sds_schema_version in your schema the suite still passes

Reply:
As discussed we agreed that the unit test did the same as the integration test and so I removed it